src/test/java/com/jcabi/github/mock/MkBranchesTest.java

```diff
@@ -0,0 +1,105 @@
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the jcabi.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.github.mock;
+
+import com.jcabi.github.Branch;
+import com.jcabi.github.Coordinates;
+import java.io.IOException;
+import java.util.Iterator;
+import javax.json.Json;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test case for {@link MkBranches}.
+ *
+ * @author Chris Rebert (github@rebertia.com)
+ * @version $Id$
+ */
+public final class MkBranchesTest {
+    /**
+     * Key for key-value pair for specifying name of repository when
+     * creating new repository.
+     */
+    private static final String REPO_NAME_PARAM = "name";
+
+    /**
+     * MkBranches should be able to create a new branch.
+     * @throws IOException if there is any I/O problem
+     */
+    @Test
+    public void createsBranch() throws IOException {
+        final String name = "my-new-feature";
+        final String sha = "590e188e3d52a8da38cf51d3f9bf598bb46911af";
+        final String user = "johnson";
+        final String repo = "my-repo";
+        final Branch branch = ((MkBranches) (new MkGithub(user).repos().create(
+            Json.createObjectBuilder()
+                .add(REPO_NAME_PARAM, repo)
+                .build()
+        ).branches())).create(name, sha);
+        MatcherAssert.assertThat(branch.name(), Matchers.equalTo(name));
+        MatcherAssert.assertThat(branch.commit().sha(), Matchers.equalTo(sha));
+        final Coordinates coords = branch.commit().repo().coordinates();
+        MatcherAssert.assertThat(coords.user(), Matchers.equalTo(user));
+        MatcherAssert.assertThat(coords.repo(), Matchers.equalTo(repo));
+    }
+
+    /**
+     * MkBranches should be able to iterate over the repo's branches.
```

Comment:
@cvrebert let's change to MkBranches can ... according to http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names